### ISSUE TITLE
Add protocol list to sheet overview

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-server#1111]
@benedikt.obermayer noted that the "Study Protocol URI" parameter is not displayed anywhere. This is parsed in import/export, but as it's a part of the study and not the tables themselves, it's not currently displayed.

The easy way to implement this is to simply list the study protocols in the Overview tab.

**TBD:** Other study-specific fields to display which are currently not?

**TBD:** Priority/milestone?

**Comment by sellth** [comment]
Maybe worth considering: Use this overview to display information contained in the "Study Protocol Description" section. The official ISA-tab examples use this field to include the entire experimental protocol. 

https://gist.github.com/c3c4306af5916856c607/download

Future perspective: Make this into a full-grown protocol section editor. Adding/removing protocols & protocol parameters could be automatically propagated to samplesheets.

**Comment by mikkonie** [comment]
Yes, the study protocol description section info should most likely be included here.

In the future there will be investigation/study/assay/protocol editing, but that's a different topic..
